winterfell/controllers/adminController.js

```diff
@@ -112,7 +112,7 @@ module.exports = function (app) {
     });
   }
 
-  app.post('/admin/reset-test-users', function (req, res) {
+  app.get('/admin/reset-test-users', function (req, res) {
```

Comment:
I know there's no body in this request so we can get away with a `GET`, but I think it's good practice to differentiate the notion between retrieving things vs. changing things.
http://stackoverflow.com/questions/504947/when-should-i-use-get-or-post-method-whats-the-difference-between-them

So since this endpoint is potentially changing users, let's keep it as `POST`